bin/configs/python-experimental.yaml

```diff
@@ -1,6 +1,6 @@
 generatorName: python-experimental
 outputDir: samples/openapi3/client/petstore/python-experimental
 inputSpec: modules/openapi-generator/src/test/resources/3_0/python-experimental/petstore-with-fake-endpoints-models-for-testing-with-http-signature.yaml
-templateDir: modules/openapi-generator/src/main/resources/python
```

Comment:
Python-experimental uses some python templates to generate python code and mostly uses the templates one level deeper.
Why have you changed this?

Reply:
This was just misunderstanding on my part.  That it's overridden within the generator makes perfect sense.

Reply:
So can you revert this change ;p?

Reply:
I saw this came back in along with the black changes - this was easy, but took some doing to get the diff happy on black.